src/Styleguide/Pages/Artist/Routes/AuctionResults/AuctionResultItem.tsx

```diff
@@ -13,32 +16,61 @@ import { Responsive } from "Styleguide/Utils/Responsive"
 import { Subscribe } from "unstated"
 import { AuctionResultsStateContainer } from "./AuctionResultsState"
 
-export interface Props {
+export interface Props extends ContextProps {
   auctionResult: AuctionResultItem_auctionResult
+  mediator?: {
+    trigger: (action: string, config: object) => void
+  }
 }
 
-export const AuctionResultItem: React.SFC<Props> = (props: Props) => {
-  return (
-    <Row>
-      <Responsive>
-        {({ xs, sm, md }) => {
-          if (xs) {
-            return <ExtraSmallAuctionItem {...props} />
-          } else if (sm || md) {
-            return <SmallAuctionItem {...props} />
-          } else {
-            return <LargeAuctionItem {...props} />
-          }
-        }}
-      </Responsive>
+export class AuctionResultItem extends React.Component<Props> {
+  render() {
+    return (
+      <Subscribe to={[GlobalState]}>
```

Comment:
This might be something strange but, I added the state here b/c we want to trigger the authentication modals via mediator (which is working fine).

However, `this.props.currentUser` is undefined (which it shouldn't be since everything is wrapped in `<ContextProvider>`, this should work just like `FollowArtistButton`, etc.). So, I was just taking it from state, where `state.system.currentUser` does work.

🤔 about why...`props.currentUser` doesn't work here? That should I think...I guess its fine for now to use state, but I was scratching my head.